src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -36,10 +38,11 @@ object WorkflowActor {
 case class WorkflowActor(id: WorkflowId,
                          namespace: WdlNamespace,
                          actualInputs: WorkflowCoercedInputs,
-                         backend: Backend) extends LoggingFSM[WorkflowState, WorkflowFailure] {
+                         backend: Backend) extends FSM[WorkflowState, WorkflowFailure] {
+  override val log = Logging(context.system, classOf[WorkflowActor])
   implicit val timeout = Timeout(5 seconds)
-
-  private val storeActor = context.actorOf(StoreActor.props(namespace, actualInputs))
+  val tag: String = s"WorkflowActor [UUID(${id.toString.split("-")(0)})]"
```

Comment:
I put that name property on `WorkflowDescriptor` to be able to surface it for use cases like `tag`.